packages/mds-utils/utils.ts

```diff
@@ -550,15 +549,7 @@ function moved(latA: number, lngA: number, latB: number, lngB: number) {
   return lngDiff > limit || latDiff > limit // very computational efficient basic check (better than sqrts & trig)
 }
 
-function normalizeToArray<T>(elementToNormalize: T | T[] | undefined): T[] {
-  if (elementToNormalize === undefined) {
-    return []
-  }
-  if (isArray(elementToNormalize)) {
-    return elementToNormalize
-  }
-  return [elementToNormalize]
-}
+const normalizeToArray = <T>(elementToNormalize: T | T[] | undefined): T[] => asArray(elementToNormalize ?? [])
```

Comment:
Are generics Turing-complete yet by the way?

Reply:
Yes, I believe they are.